api/auth_user_service.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"context"
 	"errors"
+	"fmt"
 	"net/http"
 	"strconv"
 
```

Comment:
:robot: OpenAI

No changes are made in this patch. Therefore, I don't have any comments on this patch.

<!-- This is an auto-generated comment by OpenAI -->